adr/ADR-1_connaisseur-bootstrap-sentinel.md

```diff
@@ -28,6 +28,8 @@ Deploy MutatingWebhookConfiguration through Helm when Connaisseur Pods are healt
 
 We chose option 1 over option 2, because it was important to us that a brief glance at Connaisseur's Namespace allows one to judge whether it is running properly. Option 3 was not chosen as the readiness status of Pods can be easily seen from the Service, whereas the health status would require querying every single Pod individually. We deemed that to be a very ugly, non-kubernetes-y solution and hence decided against it.
 
+**edit**: the outcome of this decision was further changed in [ADR-3](ADR-3_multi_notary_config.md).
```

Comment:
I would put this up in the `Status` of the ADR as "Amended in [ADR-3](ADR-3_multi_notary_config.md)" instead of in the outcome